addons/website/static/src/js/website.editor.js

```diff
@@ -1097,7 +1097,8 @@ var EditorBar = Widget.extend({
                         placement: 'auto top',
                     })
                     // Force-show popovers so users will notice them.
-                    .popover('show');
+                    .popover('show')
+                    .attr("contentEditable", true);
```

Comment:
A popover with contentEditable ?...
Remove this change, if the 'range' problem is appear we will fix later.
Thanks